service/src/main/java/org/jboss/sbomer/service/feature/sbom/model/RequestEvent.java

```diff
@@ -147,4 +149,35 @@ public RequestEvent save() {
         return this;
     }
 
+    @Override
+    public final boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null) {
+            return false;
+        }
+
+        Class<?> oEffectiveClass = o instanceof HibernateProxy
+                ? ((HibernateProxy) o).getHibernateLazyInitializer().getPersistentClass()
+                : o.getClass();
+        Class<?> thisEffectiveClass = this instanceof HibernateProxy
+                ? ((HibernateProxy) this).getHibernateLazyInitializer().getPersistentClass()
+                : this.getClass();
+
+        if (thisEffectiveClass != oEffectiveClass) {
+            return false;
+        }
+
+        RequestEvent requestEvent = (RequestEvent) o;
+        return Objects.equals(id, requestEvent.id);
+    }
+
+    @Override
+    public final int hashCode() {
+        return (this instanceof HibernateProxy)
+                ? ((HibernateProxy) this).getHibernateLazyInitializer().getPersistentClass().hashCode()
+                : getClass().hashCode();
+    }
```

Comment:
JPA Buddy says to not use `@Data` and `@EqualsAndHashCode` with JPA entities.

https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/